src/OSPSuite.Infrastructure/Services/LoggingBuilderExtensions.cs

```diff
@@ -29,14 +30,16 @@ public static ILoggingBuilder AddFile(this ILoggingBuilder builder, string logFi
                break;
          }
 
+         var levelSwitch = new LoggingLevelSwitch {MinimumLevel = serilogLevel};
          //this is a workaround to avoid having the date automatically appended to the log file name
          //(eg log_20210921.txt, when input name is simply log.txt).
          //Setting rollOnFileSizeLimit disables that, since the first log is log.txt, the next log_001.txt and so on
          //since we do not expect to ever reach the log limit (in the qualificationRunner fe), this results in one file
          //1073741824 is as far as I understand the maximum possible fileSizeLimitBytes
-         builder.AddSerilog(
+         builder.SetMinimumLevel(level).AddSerilog(
```

Comment:
Why is this line requiured? if we have ` .MinimumLevel.ControlledBy(levelSwitch)`
What is the difference?

Reply:
call removed